app/build.gradle

```diff
@@ -169,3 +169,12 @@ sourceSets {
         }
     }
 }
+
+test {
+    if (System.getenv('CONFIGS_FILE') == null) {
+        def rootDir = project.rootDir
+        environment('CONFIGS_FILE', "${rootDir}/local/config.yaml")
+    }
+
+    System.out.println("Using CONFIGS_FILE environment variable: " + System.getenv('CONFIGS_FILE'))
+}
```

Comment:
config file is optional, so there should not be a default value.

Reply:
1. Effective for testing only
2. if you don't need to use this environment variable, then exporting it additionally has no other effect
3. adding this variable can be very convenient in scenarios such as using ide and executing tests in batches.